### DISCUSSION TITLE
Estimated distance?

### DISCUSSION THREAD

**Discussion by roysmith** [VREMSoftwareDevelopment/WiFiAnalyzer#399] [Q&A] [answered]
I just installed this.  Nice app; thanks for making it available.  How is the distance to the access point estimated?  Is it just based on signal strength?

**Comment by VREMSoftwareDevelopment** [comment, marked as answer]
[How-to](https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer#how-to)  - How is distance calculated to Access Point
